### PR TITLE
[CALCITE-5572] Release Calcite 1.34.0

### PR DESCRIPTION
**Pull request by zabetak**

### CODE REVIEW

File: site/_docs/history.md [hunk 1]
```diff
@@ -65,18 +92,82 @@ other software versions as specified in gradle.properties.
...
+
 #### Web site and documentation
 {: #site-1-34-0}
--->
+
+* [<a href="https://issues.apache.org/jira/browse/CALCITE-5555">CALCITE-5555</a>]
+  Remove obsolete instructions for processing requests for new JIRA accounts
+* [<a href="https://issues.apache.org/jira/browse/CALCITE-5550">CALCITE-5550</a>]
+  Update instructions for requesting Jira account to use self-serve facility
+* CompositeOperandTypeChecker's java doc uses wrong class name
```
Comment (julianhyde) [inline site/_docs/history.md:170]: s/java doc/javadoc

File: site/_docs/history.md [hunk 2]
```diff
@@ -65,18 +92,82 @@ other software versions as specified in gradle.properties.
...
+* [<a href="https://issues.apache.org/jira/browse/CALCITE-5546">CALCITE-5546</a>]
+  Code style: Break long assignment expressions after '='
+* [<a href="https://issues.apache.org/jira/browse/CALCITE-5501">CALCITE-5501</a>]
+  `SqlToRelConverterTest.checkActualAndReferenceFiles` fails intermittently in Jenkins CI
+* Add test for [<a href="https://issues.apache.org/jira/browse/CALCITE-5524">CALCITE-5524</a>] JDBC adapter generates `LIMIT`, `OFFSET` in wrong order for Presto dialect
+* Add tests for [<a href="https://issues.apache.org/jira/browse/CALCITE-2980">CALCITE-2980</a>] Implement the `FORMAT` clause of the `CAST` operator
+* [<a href="https://issues.apache.org/jira/browse/CALCITE-5537">CALCITE-5537</a>]
+  Slow test case failures in `LatticeSuggesterTest`
+* Autostyle: Disallow space or newline before ')' in method declaration or call
+* TryThreadLocal values are now not-null by default
```
Comment (julianhyde) [inline site/_docs/history.md:161]: TryThreadLocal is not a build/test issue; should be under the 'minor enhancements' section; also, add back-ticks

File: site/_docs/history.md [hunk 3]
```diff
@@ -28,23 +28,50 @@ For a full list of releases, see
...
+Command used to generate the release notes
+git log b64cb1325cfe1a5143ea3ca534f991b6f881c3c5..ee9b80b0b68d442991dfaa142722e3488ec73e79 --format="%s" | sed 's|\[\(CALCITE-[0-9]\+\)\]|* [<a href="https://issues.apache.org/jira/browse/\1">\1</a>]\n|'git log b64cb1325cfe1a5143ea3ca534f991b6f881c3c5..ee9b80b0b68d442991dfaa142722e3488ec73e79 --format="%s" | sed 's|\[\(CALCITE-[0-9]\+\)\]|* [<a href="https://issues.apache.org/jira/browse/\1">\1</a>]\n|'
+-->
 
 #### Breaking Changes
 {: #breaking-1-34-0}
 
-As of [CALCITE-3870](https://issues.apache.org/jira/browse/CALCITE-3870),
-the default value of
+[<a href="https://issues.apache.org/jira/browse/CALCITE-3870">CALCITE-3870</a>]
```
Comment (julianhyde) [inline site/_docs/history.md:71]: it seems strange to split the description of 3870 over two paragraphs

File: site/_docs/history.md [hunk 4]
```diff
@@ -28,23 +28,50 @@ For a full list of releases, see
 Downloads are available on the
 [downloads page]({{ site.baseurl }}/downloads/).
 
-<!--
-## <a href="https://github.com/apache/calcite/releases/tag/calcite-1.34.0">1.34.0</a> / XXXX-XX-XX
+## <a href="https://github.com/apache/calcite/releases/tag/calcite-1.34.0">1.34.0</a> / 2023-03-XX
```
Comment (julianhyde) [inline site/_docs/history.md:31]: can you add a commented-out section for 1.35

File: site/_docs/history.md [hunk 5]
```diff
@@ -28,23 +28,50 @@ For a full list of releases, see
...
-This release comes x months after [1.33.0](#v1-33-0),
-contains contributions from x contributors, and resolves x issues.
+This release comes 1 month after [1.33.0](#v1-33-0),
+contains contributions from 18 contributors, and resolves 34 issues. It adds roughly 15 new functions
+in BigQuery library for handling dates, times, and timestamps and provides a fix
+([<a href="https://issues.apache.org/jira/browse/CALCITE-5522">CALCITE-5522</a>]) for a small breaking
+change in `DATE_TRUNC` function ([<a href="https://issues.apache.org/jira/browse/CALCITE-5447">CALCITE-5447</a>]),
+which was introduced accidentally in [1.33.0](#v1-33-0). Among other small improvements and fixes it
+brings a new breaking change ([<a href="https://issues.apache.org/jira/browse/CALCITE-3870">CALCITE-3870</a>])
+changing the default sub-query expansion strategy of `SqlToRelConverter`.
```
Comment (julianhyde) [inline site/_docs/history.md:41]: I don't think that 3870 needs a mention here, as it's covered below. But QUALIFY (5268) is a major new feature.

File: site/_docs/history.md (hunk 1 above)
Comment (zabetak) [reply site/_docs/history.md]: Fixed in https://github.com/apache/calcite/pull/3103/commits/12a9e0d1738fdc8fe8992f6d9441c2db74a2f769

File: site/_docs/history.md (hunk 2 above)
Comment (zabetak) [reply site/_docs/history.md]: Fixed in https://github.com/apache/calcite/pull/3103/commits/5c30df57f942b33e8675b6e543202f776a229ae8

File: site/_docs/history.md (hunk 3 above)
Comment (zabetak) [reply site/_docs/history.md:108]: Fixed in https://github.com/apache/calcite/pull/3103/commits/561070e3e0e4e232d0d4dd8d046c89c823a566d0

File: site/_docs/history.md (hunk 4 above)
Comment (zabetak) [reply site/_docs/history.md:67]: Fixed in https://github.com/apache/calcite/pull/3103/commits/88c075dbbd36b164b521f417f2eade6799a3b569

File: site/_docs/history.md (hunk 5 above)
Comment (zabetak) [reply site/_docs/history.md]: Fixed in https://github.com/apache/calcite/pull/3103/commits/c17c0c7f1c4254632d87f21b860849bea6551cdd

**Pull request closed without merging**